en/snap/preparation.md

```diff
@@ -19,5 +19,6 @@ Sign up for an Account to get your Sandbox API keys ready to test integration.
 Retrieve Sandbox mode API keys that will be used for next sections.
 </div>
 
-?>**Note:**
-Make sure to complete this preparation section to retrieve Client Key & Server Key, before proceeding.
+?>***Note***
+Complete the preparation section to retrieve *Client Key* and *Server Key*, before proceeding.
```

Comment:
Why did you remove the `:` after `Note` ? it is not even consistent with your other note styling.